com.archimatetool.reports/src/com/archimatetool/reports/html/HTMLReportExporter.java

```diff
@@ -325,9 +327,25 @@ private void writeElements(File elementsFolder, ST stFrame, List<EObject> list)
      */
     private void writeElement(File elementFile, ST stFrame, EObject component) throws IOException {
         stFrame.remove("element"); //$NON-NLS-1$
-        //frame.remove("children");
+        stFrame.remove("usedInViews"); //$NON-NLS-1$
+        stFrame.remove("modelRelations"); //$NON-NLS-1$
         stFrame.add("element", component); //$NON-NLS-1$
         
+        if (component instanceof ArchimateElement) {
+        	ArchimateElement element = ((ArchimateElement) component);
```

Comment:
Please change this to:

`if(component instanceof IArchimateElement element) {`